### ISSUE TITLE
Refactor Hero component

### ISSUE THREAD

**Issue by MaksymTaratuta55** [ramsayinnovations/ramsay-site#144]
Use the "With sign up and media content" Tailwind UI Hero section:  https://tailwindui.com/components/marketing/sections/heroes

New Hero component should use common components for texts (Typography component), for video (Video component).

Input, button, "Sign up to get notified when it’s ready" and "We care about the protection of your data. Read our Privacy Policy." should be in the own independent component. Also, it should be wrapped in `<form>`

**Comment by MaksymTaratuta55** [comment]
It is blocked by #142, #143

**Comment by MaksymTaratuta55** [comment]
Waiting for merge PRs #145 #146

**Comment by kirti75waytech** [comment]
@MaksymTaratuta55 Can you give me a brief on this issue?

**Comment by MaksymTaratuta55** [comment]
@kirti75waytech, sure. You need to refactor Hero.jsx to be like "With sign up and media content" Tailwind UI Hero section: https://tailwindui.com/components/marketing/sections/heroes
<img width="1416" alt="Screenshot 2021-01-12 at 10 58 36" src="https://user-images.githubusercontent.com/29330551/104292302-5669d900-54c5-11eb-8124-8cc50066101a.png">
Moshe told: "Do NOT worry about styling the Hero section right now.  I'll take care of that later on.  For now, I just want to create the components."

**Comment by kirti75waytech** [comment]
Okay @MaksymTaratuta55 I got it. Please confirm this, Do I also need to remove/hide the present `Hero` component?
![Uploading Screenshot 2021-01-12 at 8.43.09 PM.png…]()

**Comment by MaksymTaratuta55** [comment]
Yes, but I think texts should be ours. @thecodeclimber does it right?

**Comment by kirti75waytech** [comment]
> Yes, but I think texts should be ours. @thecodeclimber does it right?

Okay @MaksymTaratuta55

**Comment by kirti75waytech** [comment]
@MaksymTaratuta55 I have refactored the code and almost completed the task. The remaining task depends on #146 PR. As soon it get merged. I'll complete it.

**Comment by kirti75waytech** [comment]
@thecodeclimber @MaksymTaratuta55 branch #143 has been deleted, which was required for this issue. Do I need to leave the video part for now for this issue. Please can you confirm, If I need to create the video component. If so, can you please provide me the requirements?

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

Can we close this?

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, I think yes, if we won't do it for the nearest time or the task description will be changed

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55  -- great, thanks.
